boefjes/boefjes/job_handler.py

```diff
@@ -131,22 +132,38 @@ def handle(self, boefje_meta: BoefjeMeta) -> None:
 
             boefje_meta.arguments["input"] = ooi.serialize()
 
+        boefje_results: list[tuple[set, bytes | str]] = []
         boefje_meta.runnable_hash = plugin.runnable_hash
-        boefje_meta.environment = get_environment_settings(boefje_meta, plugin.boefje_schema)
 
-        logger.info("Starting boefje %s[%s]", boefje_meta.boefje.id, str(boefje_meta.id))
+        try:
+            logger.info("Getting environment settings for boefje %s[%s]", boefje_meta.boefje.id, str(boefje_meta.id))
+            boefje_meta.environment = get_environment_settings(boefje_meta, plugin.boefje_schema)
 
-        boefje_meta.started_at = datetime.now(timezone.utc)
+            logger.info("Starting boefje %s[%s]", boefje_meta.boefje.id, str(boefje_meta.id))
+            boefje_meta.started_at = datetime.now(timezone.utc)
+            boefje_results = self.job_runner.run(boefje_meta, boefje_meta.environment)
 
-        boefje_results: list[tuple[set, bytes | str]] = []
+        except SettingsNotConformingToSchema:
+            logger.exception(
+                "Error running boefje due to settings/schema mismatch %s[%s]",
+                boefje_meta.boefje.id,
+                str(boefje_meta.id),
+            )
+            boefje_results = [
+                (
+                    {"error/boefje"}, 
+                    f"Error running boefje ({boefje_meta.id}) due to settings/schema mismatch: {0}".format(json.dumps(plugin.boefje_schema)),
```

Comment:
I think it would be more useful for the user to put the actual validation error message from `SettingsNotConformingToSchema` here instead of the json schema. For this we need to change `SettingsNotConformingToSchema` to save the `validation_error` argument on the exception object.